cloudflare/provider_test.go

```diff
@@ -74,6 +74,24 @@ func testAccPreCheckLogpushToken(t *testing.T) {
 	}
 }
 
+func testAccPreCheckBYOIPPrefix(t *testing.T) {
+	if v := os.Getenv("CLOUDFLARE_EMAIL"); v == "" {
+		t.Fatal("CLOUDFLARE_EMAIL must be set for acceptance tests")
+	}
+
+	if v := os.Getenv("CLOUDFLARE_API_KEY"); v == "" {
+		t.Fatal("CLOUDFLARE_API_KEY must be set for acceptance tests")
+	}
+
+	if v := os.Getenv("CLOUDFLARE_ACCOUNT_ID"); v == "" {
+		t.Fatal("CLOUDFLARE_ACCOUNT_ID must be set for this acceptance test")
+	}
+
```

Comment:
You don't need to include these in here as we already have methods for them which can be referenced together.

```suggestion 
```